src/plays/drawing-board/shared/ColorPanel.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { setStrokeColor } from "../modules/currentStroke/slice";
+
+const COLORS = [
+  "#000000",
+  "#808080",
+  "#c0c0c0",
+  "#fffff",
+  "#FF5733 ",
+  "#ab8b04",
+  "#df73ff",
+  "#00ffff",
+  "#00ccff",
+  "#4666ff",
+  "#40e0d0",
```

Comment:
this color is repeated two times

Reply:
#40e0d0

Reply:
Can you share how the site looked prior to my play?

Reply:
https://reactplay.io/